src/main/cpp/util/BUILD

```diff
@@ -186,3 +186,9 @@ filegroup(
     srcs = glob(["*.cc"]) + glob(["*.h"]) + ["BUILD"],
     visibility = ["//visibility:public"],
 )
+
+filegroup(
```

Comment:
This filegroup is very confusing, it's called embedded_java_tools but it globs C++ code. Can you please explain with a comment and use a more meaningful name? Also, isn't this pretty much the same as the target above?

Reply:
Java tools can still be C++ :) By Java tools I mean tools used by the Java rules. I renamed the target and added a comment. 

> Also, isn't this pretty much the same as the target above?

Yes, but the BUILD file shouldn't be packaged into the java tools archive. It's easier to have 2 target IMO.